src/involvementAPI.js

```diff
@@ -0,0 +1,49 @@
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const appId = 'LpwpP44p6vm5v1pisyeV';
+
+export const newLike = async (itemId) => {
+  fetch(`${baseURL}apps/${appId}/likes`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: itemId }),
+  });
+};
+
+export const getLikes = async () => {
+  const response = await fetch(`${baseURL}apps/${appId}/likes`,
+    {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  return response.json();
+};
+
+export const getComments = async (movieID) => {
+  const response = await fetch(`${baseURL}apps/${appId}/comments?item_id=${movieID}`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  return response.json();
+};
+
+export const newComment = async (comment) => {
+  fetch(`${baseURL}apps/${appId}/comments`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(comment),
+  });
+};
+
+export const countComments = async (movieID) => {
+  const comments = await getComments(movieID);
+  if (comments.length !== undefined) {
+    return comments.length;
+  }
+  return 0;
+};
```

Comment:
Kindly look at this requirement:

![image](https://user-images.githubusercontent.com/57421823/152525745-d2a6331d-3c06-428b-96d4-4b05d1364ffd.png)

Therefore, 

- [ ] kindly please make functions to calculate your counters (comments and movies), and please don't use the API to give you those numbers.

- [ ] You will need to update your tests, so they check those functions' correct functionality.

That way we will meet that requirement and get closer to approval 💪